app.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+
+(function(){
+  angular
+  .module ("Angular ToDo App", [])
+  .controller("ToDo_App", ToDoFunction);
+ function ToDoFunction(){
+  var vm = this;
+  vm.todos = [
+    "Go to sleep",
+    "Go to sleep",
+    "Go to sleep",
+  ];
+
+
+   vm.todos = function(){
```

Comment:
On line 10 you just assigned an array of items to vm.todos and now you are assigning an (empty) function.